src/Mono.Android/Mono.Android.csproj

```diff
@@ -372,4 +373,24 @@
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
+  <PropertyGroup>
+    <ApiCompatibilityDir>../../external/xamarin-android-api-compatibility</ApiCompatibilityDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <ApiCompatibilityFiles Include="$(ApiCompatibilityDir)/v*.txt" />
+    <ApiCompatibilityFiles Include="$(ApiCompatibilityDir)/reference/*.dll" />
+  </ItemGroup>
+  <Target Name="AfterBuild" Inputs="$(TargetPath);@(ApiCompatibilityFiles)" Outputs="$(IntermediateOutputPath)CheckApiCompatibility.stamp">
+    <CheckApiCompatibility
+      ApiCompatPath="../../packages/Microsoft.DotNet.ApiCompat.5.0.0-beta.19554.1/tools/net472/"
```

Comment:
Coding convention in XML:

1. Use spaces (which you've done)
2. Two-space indents.
3. *Attributes* are "indented an extra indent".

(3) is so that if the element has an output, you can trivially distinguish the attributes from the child elements:

```xml
<FooTask
    Value1="Foo"
    Value2="Bar">
  <Output TaskParameter="Something" PropertyName="Else" />
</FooTask>
```

Reply:
Fixed.